### PR TITLE
Release Links 0.8 (Merchiston)

### PR DESCRIPTION
**Pull request by SimonJF**
Relational Lenses

Links 0.8 introduces support for Incremental Relational Lenses. See [the
paper](https://arxiv.org/pdf/1807.01948.pdf) for details.

Minor Changes

  1. Session exception handling is now more liberal, allowing variables
     to be shared between the success and failure blocks.

  2. Migrate to the Dune build system.

Bugfixes

  1. Fix non-typechecking examples.
  2. Fix type signatures for `domSetAttributeFromRef`,
     `domSetPropertyFromRef`, and `domSetStyleAttrFromRef`.
  3. Fix buggy implementation of `getValue` in `jslib`.
  4. Fix `domGetChildrenFromRef` to return only DOM elements
     instead of whitespaces

### CODE REVIEW

File: packages/links/links.0.8/opam [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+remove: [
+  [ "rm" "-f"  "%{links:bin}%/linx" ]
+  [ "rm" "-f"  "%{links:lib}%/prelude.links" ]
+  [ "rm" "-rf" "%{links:lib}%/stdlib" ]
+  [ "rm" "-rf" "%{links:lib}%/js" ]
+  [ "rm" "-rf" "%{links:share}%/examples" ]
+  [ "rm" "-f"  "%{links:etc}%/config" ]
+  [ "rm" "-f"  "%{links:doc}%/README" ]
+]
+
```
Comment (mseri) [inline packages/links/links.0.8/opam]: Are you sure the `install` and `remove` steps cannot be automated with dune?

**Comment by mseri** [comment]
The CI failure is related to `postgres` and looks independent of this pacakge

File: packages/links/links.0.8/opam (hunk 1 above)
Comment (SimonJF) [reply packages/links/links.0.8/opam]: Thanks for the suggestion. Most likely yes. We're still getting to grips with Dune's capabilities, but we're looking at streamlining our release process for the next minor release.

**Comment by mseri** [comment]
Ok, then let's merge before the switch happen

**Pull request merged**